spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/types/types.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.errorhandling
+
+import org.apache.spark.sql.Column
+
+package object types {
+  type ErrSourceColName = String
+  type JsonString = String // TODO make it more "type safe"
+
+  type ErrType = String
+  type ErrCode = Long // was string
```

Comment:
is this information important, about what it was? Is this gonna be changed in the future? Or should / can we treat it as Long from now on and forget about the past?

Reply:
Good point. Will remove it. It was there for back reference.